sympy/integrals/tests/test_meijerint.py

```diff
@@ -755,10 +755,15 @@ def test_issue_6462():
 
 def test_indefinite_1_bug():
     assert integrate((b + t)**(-a), t, meijerg=True
-        ) == -b**(1 - a)*(1 + t/b)**(1 - a)/(a - 1)
+        ).equals(-b**(1 - a)*(1 + t/b)**(1 - a)/(a - 1))
```

Comment:
It would be better to change this test answer to show what is now returned.

The old result was:
```python
In [4]: a, b, c = symbols("a b c")

In [5]: integrate((b + t)**(-a), t, meijerg=True)
Out[5]: 
               1 - a 
  1 - a ⎛    t⎞      
-b     ⋅⎜1 + ─⎟      
        ⎝    b⎠      
─────────────────────
        a - 1 
```
The new result is
```python
In [2]: integrate((b + t)**(-a), t, meijerg=True)
Out[2]: 
          1 - a 
   ⎛    t⎞      
-b⋅⎜1 + ─⎟      
   ⎝    b⎠      
────────────────
      a    a    
   a⋅b  - b 
```